### DISCUSSION TITLE
Multiple Joins and querying like

### DISCUSSION THREAD

**Discussion by iampopal** [supabase/supabase#3276] [Questions] [answered]
supabase
                        .from('companies')
                        .select('* , city:cityId(id, name)') 
                        .ilike('cities.name', '%kabul%')
                        .order('created_at').execute()
                        
I am selecting companies where city is kabul and cities is joined by cityId in companies table

**Comment by steve-chavez** [comment, marked as answer]
Try using the alias for the joined cities on the filter - `city.name` instead of `cities.name`:

```js
supabase
.from('companies')
.select('* , city:cityId(id, name)')
.ilike('city.name', '%kabul%')
.order('created_at').execute()
```
